.github/workflows/windows.yml

```diff
@@ -42,52 +42,204 @@ env:
   # clang_sys will search msys path before Program Files\LLVM
   # so we need to override this behaviour until we update clang-sys
   # https://github.com/KyleMayes/clang-sys/issues/150
-  LIBCLANG_PATH: C:\\Program Files\\LLVM\\bin
+  LIBCLANG_PATH: C:\Program Files\LLVM\bin
```

Comment:
I'm surprised this works, but it works!

Reply:
yeah, me too, it seems `\` only functions as an escape in double-quoted strings ([spec ref yaml 1.2](https://yaml.org/spec/1.2.2/#57-escaped-characters), [yaml 1.0](https://yaml.org/spec/1.0/index.html#syntax-scalar-double))